Return mock outputs when skip_outputs is set to `true` (#877)
gruntwork-io/terragrunt: README.md

}
```

> When `skip_outputs` is used with `mock_outputs`, mocked outputs will be returned without pulling in the outputs from remote states. This can be usefull when you disable the backend initialization (`remote_state.disable_init`) in a CI by example.

Reviewer: ```suggestion
When `skip_outputs` is used with `mock_outputs`, mocked outputs will be returned without pulling in the outputs from remote states. This can be useful when you disable the backend initialization (`remote_state.disable_init`) in CI for example.
```
